src/python/dxpy/asset_builder.py

```diff
@@ -202,7 +201,22 @@ def build_asset(args):
         if not args.json:
             print("Uploading input files for the AssetBuilder", file=sys.stderr)
 
-        conf_file = dx_upload(asset_conf_file, dest_project_name, dest_folder_name, args.json)
+        if args.visibility:
+            asset_conf["hidden"] = (args.visibility == 'hidden')
+        if not asset_conf.get("hidden"):
```

Comment:
This will change the default behavior of dx-toolkit, right? Currently, when building assets, it will be visible by default, with this change it will be hidden.

Should we use --visibility hidden when building the nextaur instead of changing the default behavior?

Reply:
Changed. Thanks:D